misc/patternfly-showcase.css

```diff
@@ -0,0 +1,537 @@
+.text-overflow-pf {
```

Comment:
You are overwritting ```text-overflow-pf``` for the entire application.  This needs to be limited to just ```.showcase```